scripts/dotnet-cli-build/Utils/BuildVersion.cs

```diff
@@ -13,6 +13,7 @@ public class BuildVersion
         public string VersionSuffix => $"{ReleaseSuffix}-{CommitCountString}";
         public string NuGetVersion => $"{Major}.{Minor}.{Patch}-{VersionSuffix}";
         public string NetCoreAppVersion => $"{Major}.{Minor}.{Patch}-rc2-3{CommitCountString}";
+        public string HostVersion => $"{Major}.{Minor}.1-rc2-{CommitCountString}-00";
```

Comment:
This is the version of the host nuget package as it exists. (https://dotnet.myget.org/gallery/dotnet-core)

We should unify these...

@schellap any context on why this is different?